src/rlgl.h

```diff
@@ -666,6 +666,7 @@ RLAPI void rlglInit(int width, int height);             // Initialize rlgl (buff
 RLAPI void rlglClose(void);                             // De-initialize rlgl (buffers, shaders, textures)
 RLAPI void rlLoadExtensions(void *loader);              // Load OpenGL extensions (loader function required)
 RLAPI int rlGetVersion(void);                           // Get current OpenGL version
+RLAPI GLFWglproc rlGetProcAddress(const char* procname) // Get current Proc address
```

Comment:
Thx, what about now?

Reply:
But what if i just inlcude it when this its Desktop...?

Reply:
> All these errors are because of the implicit declaration of `glfwGetProcAddress()`. To "fix" that it would be required to include the `GLFW` headers to `rlgl`, which would make all platforms depend on `GLFW` for absolutely no reason.
> 
> So, **again**, please don't merge this PR.

Or do you have any other idea to impliment it?